src/overlays/actors/ovl_En_Warp_tag/z_en_warp_tag.c

```diff
@@ -224,7 +224,7 @@ void EnWarpTag_RespawnPlayer(EnWarptag* this, PlayState* play) {
         if (new15E < 0) {
             new15E = 0;
         }
-        func_80165658(new15E * 0.04f); // unknown Play_ function
+        Play_SetMotionBlurAlpha(new15E * 0.04f);
```

Comment:
Would `/25.0f` match? or maybe `* (1.0f / 25.0f)`
```suggestion
        Play_SetMotionBlurAlpha(new15E / 25.0f);
```